extensions/wikia/BannerNotifications/BannerNotifications.setup.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * Docs are on internal in UI Style Guide
+ */
+$wgExtensionCredits['bannernotifications'][] = array(
+	'name' => 'BannerNotifications',
+	'descriptionmsg' => 'bannernotifications-desc',
+	'author' => [
+		'Hyun Lim',
+		'Maciej Brencz',
+		'Bartłomiej (Bart) Kowalczyk'
+	],
+	'url' => 'https://github.com/Wikia/app/tree/dev/extensions/wikia/BannerNotifications'
+);
+
+$dir = dirname(__FILE__) . '/';
+
+/**
+ * Classes
+ */
+$wgAutoloadClasses[ 'BannerNotificationsController' ] = $dir . 'BannerNotificationsController.class.php';
+
+/**
+ * Hooks
+ */
+// confirmations
+$wgHooks['ArticleDeleteComplete'][] = 'BannerNotificationsController::addPageDeletedConfirmation';
+$wgHooks['ArticleUndelete'][] = 'BannerNotificationsController::addPageUndeletedConfirmation';
+#$wgHooks['EditPageSuccessfulSave'][] = 'BannerNotificationsController::addSaveConfirmation'; // BugId:10129
```

Comment:
What do you think about removing this line? I don't expect it get used again when we introduced VE.